debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -389,11 +389,11 @@ public void shouldProcessNotNullColumnsConnectDateTypes() throws Exception {
     public void shouldProcessNotNullColumnsAdaptiveDateTypes() throws Exception {
         final Struct before = testProcessNotNullColumns(TemporalPrecisionMode.ADAPTIVE);
         if (before != null) {
-            assertThat(before.get("created_at")).isEqualTo(0L);
+            assertThat(before.get("created_at")).isEqualTo("1970-01-01T00:00:00Z");
```

Comment:
@ismailsimsek How is it possible that this has changed? This would break backward compatibility.

Reply:
fixed